### DISCUSSION TITLE
sometimes obfuscate failure by-b 'disk-number'

### DISCUSSION THREAD

**Discussion by ApolloHY** [dashingsoft/pyarmor#1264] [Q&A]
INFO     pyarmor gen -O obfdist -b "NA4BTARY" -a xt.py
INFO     Python 3.9.13
INFO     Pyarmor 8.2.1 (pro), 005121, non-profits
INFO     Platform windows.x86_64
INFO     search inputs ...
INFO     find script xt.py
INFO     find 1 top resources
ERROR    invalid device info "NA4BTARY"
the same case with disk-number:0000_0000_0000_0000_8CE3_8E03_005B_F21D.
and also obfuscate failure by -b 'mac_address' for example:B0-6E-BF-D0-EA-8C
ERROR    invalid device info "B0-6E-BF-D0-EA-8C"
i can do this by -b 'B0:6E:BF:D0:EA:8C'
but can;'t start: RuntimeError: this license key is not for this machine (1:10185)

**Comment by jondy** [comment]
It's better to use hardware information got by `pyarmor-7 hdinfo`

**Comment by ApolloHY** [comment]
by this way,i only can used in terminate,how to get it this way on someone else's computer?he doesn't have python
